src/components/metadata/styles/metadata.scss

```diff
@@ -0,0 +1,234 @@
+@import '../../../styles/variables';
+@import '../../../styles/mixins';
+
+.pipeline-metadata {
+  @include sidebarBase;
+
+  position: absolute;
+  top: -1px; /* Avoids pixel rounding gaps */
+  right: 0;
+  bottom: -1px;
+  z-index: 2;
+  width: 100%;
+  min-width: $metadata-sidebar-min-width-open;
+  max-width: $metadata-sidebar-width-open;
+  padding: 6px 0 24px 0;
+  overflow-y: auto;
+  transform: translateX(100%);
+  transition: transform ease 0.4s;
+
+  &--visible {
+    transform: translateX(0);
+  }
+
+  @media print {
+    display: none;
+  }
+
+  .kui-theme--light & {
+    background: $color-bg-light-3;
+  }
+
+  .kui-theme--dark & {
+    background: $color-bg-dark-6;
+  }
+}
+
+.pipeline-metadata__header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.pipeline-metadata__close-button {
+  flex-grow: 0;
+  flex-shrink: 0;
+  margin: 0.25em -8px 0 0.8em;
+
+  &,
+  & svg {
+    width: 24px;
+    height: 24px;
+  }
+}
+
+.pipeline-metadata__header,
+.pipeline-metadata__label,
+.pipeline-metadata__row {
+  margin: 1.9em 36px;
+  font-size: 1.48em;
+}
+
+.pipeline-metadata__icon {
+  display: inline-block;
+  flex-grow: 0;
+  flex-shrink: 0;
+  width: 30px;
+  height: 30px;
+  margin: 0 0.85em 0 -0.3em;
+
+  .kui-theme--light & {
+    fill: $color-text-light;
+  }
+
+  .kui-theme--dark & {
+    fill: $color-text-dark;
+  }
+}
+
+.pipeline-metadata__title {
+  flex-grow: 1;
+  margin: 0;
+  font-weight: normal;
+  font-size: 1.38em;
+  line-height: 1.6;
+  word-break: break-word;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.8);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.8);
+  }
+}
+
+.pipeline-metadata__list {
+  margin: 0;
+}
+
+.pipeline-metadata__label {
+  margin-bottom: 0.7em;
+  font-weight: normal;
+  user-select: none;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.55);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.55);
+  }
+}
+
+.pipeline-metadata__row {
+  margin-top: 0.7em;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.55);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.55);
+  }
+}
+
+.pipeline-metadata__value {
+  display: inline;
+  flex-grow: 0;
+  margin-right: auto;
+  line-height: 1.5;
+  word-break: break-word;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.8);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.8);
+  }
```

Comment:
I changed these to use `0.85` is that what you meant?

Reply:
That's better, but what I meant was, why do they need reduced opacity at all? I'm not sure that it improves things. But this goes against what I was saying in the meeting we just had, about quibbling over small design details. Personally I'd be inclined to make it 100% opacity (for consistency and also readability) if I were designing it, but I think it's better to just leave it and we can revisit if necessary

Reply:
Yeah the approach to colours in both code and design on this I'm finding to be pretty confusing, so apologies for mixing any up. I agree it would be a good idea to consolidate and improve the approach in a later update rather than blocking here?

Reply:
Totally agree. Let's park this for now and discuss later if necessary.